core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -105,6 +105,10 @@
                                   style="margin-left: 10px;">
         </exploration-summary-tile>
       </div>
+
+      <div class="conversation-skin-back-to-collection-container">
```

Comment:
You'll need some logic to ensure that this was accessed via collections because the button will also show when the user finishes a single exploration and be very confused.